tagger.py

```diff
@@ -0,0 +1,55 @@
+import os
+import pydicom
+import argparse
+
+
+class TagError(Exception):
+    pass
+
+
+def add_send_tag(dicomfile, group, identifier, tag_value):
```

Comment:
I like how `tagger.py` is generic; the only minor outlier is `send_tag` in this function name.